### ISSUE TITLE
'Description' field

### ISSUE THREAD

**Issue by ckolderup** [rauchg/slackin#181]
It'd be nice if we could have a configurable description field via an environment variable where we could write a short description of the purpose of the slack team. I admin a medium-sized slack team that has a fairly narrow focus and some people get invited to it by their friends, who then have to spend some time explaining what it's all about and why they should join. If we could write a small 'mission statement' in a place that's more obvious than behind the Code of Conduct link, it would be more immediately obvious to people if it's something they're interested in.

**Comment by laughinghan** [comment]
You can set a topic and a purpose in a Slack channel ( https://get.slack.help/hc/en-us/articles/201654083-Setting-a-channel-topic-or-purpose ) but that's outside the scope of this project, which is just the signup process. Are you talking about having a description on the signup page, so before people enter their email to get an invite they see this description? You could edit/submit a patch to this file, which uses a virtual DOM to generate the HTML for the signup page: https://github.com/rauchg/slackin/blob/master/lib/splash.js

**Comment by ckolderup** [comment]
@laughinghan yep, the latter is what I meant. I'll see what I can do—don't have much experience with this stack but I'll try to submit a PR soon.

**Comment by forcevive** [comment]
Hi @ckolderup have you figured this out yet? I'm looking for a way to configure the text on the sign-up page, too. A paragraph to explain what Slack is and what we do there ... many people still don't know Slack yet. 

http://integral-entrepreneurship.herokuapp.com

**Comment by ckolderup** [comment]
I haven't had a chance, no-- I've had some life and work changes that I've had to deal with. No estimate on when I'd have time for hobby coding, so if you or someone else is able to take a stab at it, you wouldn't be duplicating any effort.

**Comment by forcevive** [comment]
Got it. Unfortunately I’m not a coder … was following the (excellent) instructions to get Slackin to run.
